ext-antora/comp-version.js

```diff
@@ -1,14 +1,19 @@
 'use strict'
 
-// Extension to print the component + version that will be processed 
+// Extension to print the component, version and number of files that will be processed 
 
 module.exports.register = function () {
   this.once('contentAggregated', ({ contentAggregate }) => {
-    console.log('\nProcessing the following components and versions\n')
+    console.log('\nProcessing the following components, versions and number of files\n')
+    var total_files = 0
     const component_table = []
       contentAggregate.forEach((bucket) => {
         component_table.push ({Name: bucket.name, Version: bucket.version || '~', Files: bucket.files.length})
+        total_files += parseInt(bucket.files.length)
```

Comment:
I suppose that `bucket.files.length` returns the "length" of the list of files (the number of files), and not the total size (length) of the content of the files.

Reply:
Correct, number of files, not file lenght.